packages/ember-router-helpers/rollup.config.js

```diff
@@ -0,0 +1,49 @@
+import babel from '@rollup/plugin-babel';
+import { Addon } from '@embroider/addon-dev/rollup';
+
+const addon = new Addon({
+  srcDir: 'src',
+  destDir: 'dist',
+});
+
+export default {
+  // This provides defaults that work well alongside `publicEntrypoints` below.
+  // You can augment this if you need to.
+  output: addon.output(),
+
+  plugins: [
+    // These are the modules that users should be able to import from your
+    // addon. Anything not listed here may get optimized away.
+    addon.publicEntrypoints(['helpers/**/*.js', 'utils/**/*.js']),
+
+    // These are the modules that should get reexported into the traditional
+    // "app" tree. Things in here should also be in publicEntrypoints above, but
+    // not everything in publicEntrypoints necessarily needs to go here.
+    addon.appReexports(['helpers/**/*.js', 'utils/**/*.js']),
```

Comment:
```suggestion
    addon.appReexports(['helpers/**/*.js']),
```

Reply:
As utils are used in tests I'm not sure
https://github.com/concordnow/ember-router-helpers/blob/master/packages/test-app/tests/integration/helpers/route-params-test.js#L6

Reply:
Fixed
https://github.com/rwjblue/ember-router-helpers/pull/293/commits/d057f585960c5e244b1aa00e0f7030c43d26a963